mvp-spec/versioning.md

```diff
@@ -0,0 +1,35 @@
+# Versioning
```

Comment:
IMO, this document in its current state is a bit premature and should be done in another PR.

Reply:
it has nothing to do in this PR... fail! 🤦‍♂️
Sorry about that.